diff --git a/dist/maplibre-ui-translations.js b/dist/maplibre-ui-translations.js
index 54d5c50..dc6a4d1 100755
--- a/dist/maplibre-ui-translations.js
+++ b/dist/maplibre-ui-translations.js
@@ -1,4 +1,30 @@
-import { defaultLocale } from 'maplibre-gl/src/ui/default_locale';
+const defaultLocale = {
+    'AttributionControl.ToggleAttribution': 'Toggle attribution',
+    'AttributionControl.MapFeedback': 'Map feedback',
+    'FullscreenControl.Enter': 'Enter fullscreen',
+    'FullscreenControl.Exit': 'Exit fullscreen',
+    'GeolocateControl.FindMyLocation': 'Find my location',
+    'GeolocateControl.LocationNotAvailable': 'Location not available',
+    'LogoControl.Title': 'MapLibre logo',
+    'Map.Title': 'Map',
+    'Marker.Title': 'Map marker',
+    'NavigationControl.ResetBearing': 'Drag to rotate map, click to reset north',
+    'NavigationControl.ZoomIn': 'Zoom in',
+    'NavigationControl.ZoomOut': 'Zoom out',
+    'Popup.Close': 'Close popup',
+    'ScaleControl.Feet': 'ft',
+    'ScaleControl.Meters': 'm',
+    'ScaleControl.Kilometers': 'km',
+    'ScaleControl.Miles': 'mi',
+    'ScaleControl.NauticalMiles': 'nm',
+    'GlobeControl.Enable': 'Enable globe',
+    'GlobeControl.Disable': 'Disable globe',
+    'TerrainControl.Enable': 'Enable terrain',
+    'TerrainControl.Disable': 'Disable terrain',
+    'CooperativeGesturesHandler.WindowsHelpText': 'Use Ctrl + scroll to zoom the map',
+    'CooperativeGesturesHandler.MacHelpText': 'Use ⌘ + scroll to zoom the map',
+    'CooperativeGesturesHandler.MobileHelpText': 'Use two fingers to move the map',
+};
 import { fr } from './fr';
 import { es } from './es';
 import { de } from './de';
